modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,9 +90,19 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Import SAML Metadata
+
+You can import your identity provider's SAML 2.0 metadata to Anypoint Platform using the *Import IdP Metadata* link. Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form. Note that you can import the SAML 2.0 metadata only when you are creating a new SAML 2.0 configuration.
+
+When you upload an XML file to import your metadata:
+
+* The file must be smaller than 16 KB.
+* If you upload an XML file that contains more than entry, Anypoint Platform uses the first identity provider configuration.
+* Anypoint Platform does not check the validity of the values in the XML that you upload. You must ensure your identity configuration is correct before creating the identity provider configuration.
```

Comment:
If you keep this third item as a bullet, remember to apply our bullet list formatting standard:

```suggestion
* Anypoint Platform does not check the validity of the values in the XML that you upload. +
You must ensure that your identity configuration is correct before creating the identity provider configuration.
```
Or maybe this is another prerequisite; is this before or after (verify rather than ensure that)?

Reply:
```suggestion
* Anypoint Platform does not check the validity of the values in the XML that you upload. +
Verify that your identity configuration is correct before creating the identity provider configuration.
```